controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -437,6 +437,13 @@ columns:
     filter: ifdirections
     default: false
     width: 10
+  - id: Udns
+    name: interface assigned UDN label
```

Comment:
Could it be simply named "UDN name" ? Or "UDN label", if "name" isn't accurate ? Just trying to get a more user friendly name for the column.

Reply:
UDN Labels is fine